package.json

```diff
@@ -4,7 +4,9 @@
   "description": "",
   "main": "bash.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node app",
+    "seed": "node seed"
```

Comment:
It was a requirement (I am presuming you forgot) to add these scripts. Try adding a script `start-dev` which launches the server but restarts it if you change the server code (hint: use `nodemon`).